content/en/contribute/code/workflow.md

```diff
@@ -110,13 +110,13 @@ The main branch is `main` (or `master`) which must be kept stable so as not to i
 
 The commit format should follow this [conventional-changelog angular preset](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular). This means we can automate some of the release process. Examples are provided below.
 
-Type | Example commit message | Release type
--- | -- | --
-Bug fixes | fix(#123): infinite spinner when clicking contacts tab twice | patch
-Performance | perf(#789): lazily loaded angular modules | patch
-Features | feat(#456): add home tab | minor
-Non-code | chore(#123): update README | none
-Breaking| perf(#2): remove reporting rates feature <br/> BREAKING CHANGE: reporting rates no longer supported | major
+Type | Description | Example commit message | Release type
+-- | -- | -- | --
+Bug fixes | A bug fix. | fix(#123): infinite spinner when clicking contacts tab twice | patch
+Performance | A code change that improves performance. Measure the performance improvement to inform the community. | perf(#789): lazily loaded angular modules | patch
+Features | A new feature or change in the code that users will notice. | feat(#456): add home tab | minor
+Non-code | A change that user won't notice, like a change in a README file, adding e2e tests, update dependencies, removing unused code, etc. | chore(#123): update README | none
+Breaking | Breaking changes should be explained under performance type. | perf(#2): remove reporting rates feature <br/> BREAKING CHANGE: reporting rates no longer supported | major
```

Comment:
"Breaking" is special because it could apply to any of the above types. "perf" was just this example, but it's highly likely to be a "feat" but could even be a "fix". Maybe we need to split it out of the table to make this clear?

Reply:
I've created a note and added that piece there. What do you think?